.github/workflows/on-push-to-main-branch.yml

```diff
@@ -0,0 +1,24 @@
+name: On push to main branch
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
+
+      - name: Generate README
+        uses: momentohq/standards-and-practices/github-actions/generate-and-commit-oss-readme@gh-actions-v1
+        with:
+          project_status: official
+          project_stability: alpha
+          project_type: sdk
+          sdk_language: Python
+          usage_example_path: ./examples/example.py
```

Comment:
missing newline here

Reply:
fixed in https://github.com/momentohq/client-sdk-python/pull/196/commits/6123929ac3f773e51815231422a3cdf3dd9b5af5